src/components/member/BusinessDetails/BusinessDetails.jsx

```diff
@@ -239,21 +230,54 @@ const BusinessDetailsList = (props) => {
             if (id) {
               return (
                 <Container key={id} className="business-details-list__container">
-                  <BusinessDetailsPicture
-                    id={id}
-                    imageUrl={imageUrl}
-                    onClickBusinessProfileImage={onClickBusinessProfileImage}
-                    businessCategory={businessCategory}
-                  />
-                  <BusinessDetailsContent
-                    name={name}
-                    description={description}
-                    phone={phone}
-                    products={products}
-                    registeredAddress={registeredAddress}
-                    services={services}
-                    website={website}
-                  />
+                  <div className="col-md-4 col-sm-6 business-box-container business-box-01-container">
+                    <div className="business-circle">
+                      <a onClick={onClickBusinessProfileImage}><IconCamera /></a>

```

Comment:
Use Hyperlink component
```suggestion
                      <Hyperlink onClick={onClickBusinessProfileImage}><IconCamera /></a>

```

Reply:
Done